omero2pandas/upload.py

```diff
@@ -131,11 +230,26 @@ def create_table(df, table_name, parent_id, parent_type, conn, chunk_size):
         link_obj.link(target_obj, annotation)
         link_obj = conn.getUpdateService().saveAndReturnObject(
             link_obj, _ctx={"omero.group": str(parent_group)})
-        LOGGER.info("Saved annotation link")
-
+        annotation_id = link_obj.child.id.val
+        LOGGER.info(f"Uploaded as FileAnnotation {annotation_id}")
+        extra_link_objs = []
+        unloaded_ann = omero.model.FileAnnotationI(annotation_id, False)
```

Comment:
Is there a reason we are not importing `FileAnnotationI` into the package scope?